R/create_query_data.R

```diff
@@ -817,13 +819,29 @@ assert_terms <- function(terms,
 basket_select <- function(name = NULL,
                           id = NULL,
                           scope = NULL,
-                          type) {
-  out <- list(
-    name = name,
-    id = id,
-    scope = scope,
-    type = type
-  )
+                          type,
+                          ...) {
+  args <- eval(substitute(alist(...)))
```

Comment:
Do we have any asset functions that we could use here? https://pharmaverse.github.io/admiraldev/dev/reference/assert_expr_list.html

????

Reply:
Hmm, I guess you could pass in anything you want though, as long as you read it in correctly in the user-defined `get_terms` function.  There is a check there for named parameters.

Reply:
okay if you think that is enough sanitization on inputs and information for users if something goes wrong.